src/pages/signup.js

```diff
@@ -199,6 +214,18 @@ const Signup = () => {
           </Stack>
         </Form>
       </Column>
+      <Modal
+        open={open}
+        passiveModal
+        
+        onRequestClose={() => navigate("/login")}
+      >
+        <Stack gap={7}>
+              <h4>Successfull signed in!</h4>
+              <p> You will be notified when your acccount is approved </p>
+              
```

Comment:
Also, please use the prettier vscode extension to format your code. You can also set it to automatically format your files when you save :)